src/main/java/duke/helper/TaskList.java

```diff
@@ -0,0 +1,41 @@
+package duke.helper;
+
+import duke.tasks.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> taskList = new ArrayList<>();
+
+
+    public TaskList(){
+        this.taskList = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+
+    public Task get(int i){
```

Comment:
Maybe can rename this to getTask 
```suggestion
    public Task getTask(int i) {
```